### DISCUSSION TITLE
Release of logisim V3.9.0

### DISCUSSION THREAD

**Discussion by BFH-ktt1** [logisim-evolution/logisim-evolution#1869] [Q&A]
Hi all,

I think it is about time to release a new version. My question to you all is what should still be merged before the release, and what would be a good date.

Thanks for your feedback!

**Comment by juli-99** [comment]
The two PRs (#1861, #1870) that introduce translations for the newly introduced shortcut settings should definitely be merged.

**Reply by MarcinOrlowski** [reply]
+1 for translations. As for date - 3.8.0 went public 2022-10-02 and if I remember correctly you always wanted it to be in sync with start of academic school year, so why not keep that this time too and simply keep October 2 again (which is Monday). Just push RC1 now and we will have 2 weeks to polish any sharp edges. I'd also like to review the PL translation one more time as I got some concerns about it in certain areas.

**Comment by R3dst0ne** [comment]
When 3.9.0 gets released, the references mentioned in #1534 have to be updated again.

In addition, it would be the perfect time to finaly fix the [snap package](https://snapcraft.io/logisim-evolution).
> The package name is logisim-evolution and the name of the organization is Logisim-Evolution.
Please fix this if possible.
https://github.com/logisim-evolution/logisim-evolution/issues/1647#issuecomment-1464101270

Now that the nightly builds work, this should be possible
> Push new snaps to latest/edge channel automatically
https://github.com/logisim-evolution/logisim-evolution/issues/1647#issuecomment-1453502625
